app/views/crops/_photos.html.haml

```diff
@@ -1,5 +1,15 @@
+%h2 Photos of #{crop.name} harvests
+.row
+  - unless crop.harvest_photos.empty?
```

Comment:
Should we have some "nothing to show here" text if `harvest_photos` is empty? I guess it's consistent with what's there already, so if we're going to change it we should do so separately.